webdriver-spec.html

```diff
@@ -6292,30 +6292,34 @@
     to <var>primary</var>, <code>pressed</code> set to an empty set
     and both <code>x</code> and <code>y</code> set to <code>0</code>.
 
-    <p>The <dfn>corresponding input state type</dfn> for a
-    label <var>action type</var> is given by the following table:
+<p>The <dfn>corresponding input state type</dfn>
+ for a label <var>action type</var> is given by the following table:
 
-      <table class=simple>
-        <thead>
-          <tr>
-            <th><var>action type</var>
-            <th>Corresponding input state type
-        </thead>
-        <tr>
-          <td><code>"none"</code>
-          <td><a>null input state</a>
-        </tr>
-        <tr>
-          <td><code>"key"</code>
-          <td><a>key input state</a></td>
-        </tr>
-        <tr>
-          <td><code>"pointer"</code>
-          <td><a>pointer input state</a></td>
-        </tr>
-      </table>
+<table class=simple>
+ <thead>
+  <tr>
+   <th><var>Action type</var>
+   <th>Input state
+  </tr>
+ </thead>
+
+ <tr>
+  <td>"<code>none</code>"
```

Comment:
Because it is otherwise unclear if the quotes are part of the string, and it also helps set strings apart from property names, for which we also use `<code>`.